test/dotcom_web/controllers/transit_near_me_controller_test.exs

```diff
@@ -161,42 +99,77 @@ defmodule DotcomWeb.TransitNearMeControllerTest do
 
   describe "with valid location params" do
     test "assigns stops with routes", %{conn: conn} do
-      params = %{"address" => %{"latitude" => "valid", "longitude" => "valid"}}
+      stops = build_list(3, :stop_item)
 
-      conn = get(conn, transit_near_me_path(conn, :index, params))
+      expect(LocationService.Mock, :reverse_geocode, fn latitude, longitude ->
+        assert latitude == @latitude
+        assert longitude == @longitude
+        {:ok, [@located_address]}
+      end)
 
-      assert conn.status == 200
+      expect(MBTA.Api.Mock, :get_json, 3, fn "/stops/", opts ->
```

Comment:
Is there a reason we're mocking the `MBTA.Api` (which is higher up) than the `Stops.Repo` here?

Reply:
....huh, that _is_ weird.

Reply:
It's populated via `TransitNearMe.build/2`, which calls `Stops.Nearby.nearby_with_varying_radius_by_mode/1`, which calls `MBTA.Api.Stops.all()` directly. Bananas!

Reply:
Might be making TNM slower than it needs to be.